range_pool.go

```diff
@@ -55,36 +55,16 @@ const (
 type Config struct {
 	// Dependencies.
 	Logger  micrologger.Logger
-	Storage microstorage.Service
+	Storage microstorage.Storage
 }
 
 // DefaultConfig provides a default configuration to create a new range pool by
 // best effort.
 func DefaultConfig() Config {
-	var err error
-
-	var newLogger micrologger.Logger
-	{
-		config := micrologger.DefaultConfig()
-		newLogger, err = micrologger.New(config)
-		if err != nil {
-			panic(err)
-		}
-	}
-
-	var newStorage microstorage.Service
-	{
-		config := microstorage.DefaultConfig()
-		newStorage, err = microstorage.New(config)
-		if err != nil {
-			panic(err)
-		}
-	}
```

Comment:
Creating a memory storage and leaving it like that by mistake felt so bad to me. Thanks!